schemas/fingerprint-server-api.yaml

```diff
@@ -1426,6 +1426,10 @@ components:
           type: boolean
           description: VPN or other anonymizing service has been used when sending the request.
           example: false
+        originTimezone:
```

Comment:
Can you also add it to the [examples](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/tree/main/examples)?

Reply:
Thanks, added